### PR TITLE
uiplib/scrape.py: Now all the images are successfully downloaded

### LINKED ISSUE #88: Not all images are downloaded
The script downloads image only from the first page in which only few "hrefs" end in "jpg,png..etc". Hence out of the first 25 links only around 4-5 images are downloaded.

### PR DESCRIPTION
**Pull request by teja-315**
Fixes #88

### CODE REVIEW

File: uiplib/scrape.py [hunk 1]
```diff
@@ -36,23 +36,33 @@ def get_images(url):
...
+
+        for link in thumbnails:
+            if link['href'].endswith(('jpg', 'png', 'jpeg', 'bmp')):
+                image_links.append(link['href'])
+            if(len(image_links) == NUMBER_OF_IMAGES_TO_PARSE):
+                break
+        # This will store the first "a" tag with attribute rel = "nofollow
+        # next" to p_url
+        p_url = soup.find("a", attrs={"rel" : "nofollow next"})
+        # This will get the value of attribute 'href' in p_url
```
Comment (nkprince007) [inline uiplib/scrape.py]: You could leave a better comment here explaining that it fetches DOM from next page or so.

Comment (abhsag24) [inline uiplib/scrape.py]: yes please don't make it so technical that's why we have comments in the first place :P

File: uiplib/scrape.py [hunk 2]
```diff
@@ -36,23 +36,33 @@ def get_images(url):
...
+        p_url = p_url.attrs['href']
 
-    for link in thumbnails:
-        if link['href'].endswith(('jpg', 'png', 'jpeg', 'bmp')):
-            image_links.append(link['href'])
-        if(len(image_links) == NUMBER_OF_IMAGES_TO_PARSE):
-            break
+    for link in image_links:
+        print(link)
```
Comment (abhsag24) [inline uiplib/scrape.py]: do we need this link or did you use this for debugging?

**Comment by abhsag24** [comment]
@teja-315 left a review :D you're going awesome :P

**Comment by abhsag24** [comment]
oops sorry closed by mistake :D

**Comment by abhsag24** [comment]
hey @teja-315 we moved your commit to this PR: https://github.com/NIT-dgp/UIP/pull/130 to merge it! :D

**Pull request closed without merging**